examples/dart/test/read_write_data_test.dart

```diff
@@ -83,4 +83,19 @@ void main() {
     expect(prius!.miles, 500);
     cleanUpRealm(realm);
   });
+
+  test('Write async', () async {
+    final config = Configuration.local([Car.schema]);
+    final realm = Realm(config);
+    // :snippet-start: write-async
+    // Add Subaru Outback to the realm using `writeAsync`
+    Car newOutback = Car("Subaru", model: "Outback Touring XT", miles: 2);
+    await realm.writeAsync(() {
```

Comment:
```suggestion
   realm.writeAsync(() {
```

you actually don't want to use `await` here b/c that makes the operation thread blocking.